st2api/st2api/app.py

```diff
@@ -68,6 +69,8 @@ def setup_app(config={}):
     app = router.as_wsgi
 
     # Order is important. Check middleware for detailed explanation.
+    app = StreamingMiddleware(app, path_whitelist=['/v1/executions/*/output*',
+                                                  '/v1/executions/*/stderr*'])
```

Comment:
These endpoints correspond to changes in the the single output collection in the database?

Reply:
Good catch - still need to remove /stderr one.

Reply:
c18d9ee476c67d2194364ee9fea20ddeb0d767d9.